code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1023,19 +1023,61 @@
 /datum/reagent/minttoxin
 	name = "Mint Toxin"
 	id = MINTTOXIN
-	description = "Useful for dealing with undesirable customers."
+	description = "Useful for dealing with undesirable customers. The undiluted version of Mint Extract."
 	reagent_state = REAGENT_STATE_LIQUID
 	color = "#CF3600" //rgb: 207, 54, 0
 	density = 0.898
 	specheatcap = 3.58
+	custom_metabolism = 0.01 //so it lasts 10x as long as regular minttox
+	var/fatgokaboom = TRUE
+	nutriment_factor = 2.5 * REAGENTS_METABOLISM //about as nutritious as sugar
+	sport = SPORTINESS_SUGAR //a small performance boost from being COOL AND FRESH
+	var/chillcounter = 0
 
 /datum/reagent/minttoxin/on_mob_life(var/mob/living/M, var/alien)
 
 	if(..())
 		return 1
 
-	if(M_FAT in M.mutations)
+	if(prob(5))
+		to_chat(M, "<span class='notice'>[pick("You feel minty fresh!","If freshness could kill you'd be a serial killer!","You feel the strange urge to share this minty freshness with others!","You have a sudden craving to drink ice cold water.","Ahh, so refreshing!")]</span>")
+
+	if(M.bodytemperature > 310) //copypasted from the cold drinks check so I don't have to change minttox internally and maybe most certainly break shit in the process
+		M.bodytemperature = max(310, M.bodytemperature + (-5 * TEMPERATURE_DAMAGE_COEFFICIENT)) //that minty freshness my dude, chill out
+
+	if(fatgokaboom && M_FAT in M.mutations)
 		M.gib()
+		
+	if(ishuman(M))
+	var/mob/living/carbon/human/H = M
```

Comment:
missed indent

Reply:
that's how it is on other parts of the code though

Reply:
It's definitely not, considering this doesn't compile